sysutils/istioctl/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                istioctl
+
+github.setup        istio istio 1.0.2
+categories          sysutils
+platforms           darwin
+supported_archs     x86_64
+license             Apache-2
+
+maintainers         {vmware.com:nnikolay} openmaintainer
+
+description         Istio configuration utility
+long_description    Create, list, modify, and delete configuration \
+                    resources in the Istio system.
+
+set goproj          istio.io/${github.project}
+
+depends_build       port:go
+use_configure       no
+worksrcdir          src/${goproj}
+
+checksums           rmd160  b6c61241b682a1b7f5676de60f3a4387bc057967 \
+                    sha256  12dc01cff8ebb335a08c62c9da95f9c71364d9dfab9c0c0baa5245f17202b057 \
+                    size    18824606
+
+post-extract {
+    xinstall -d ${workpath}/src/istio.io
+    move ${workpath}/${github.project}-${github.version} \
+        ${worksrcpath}
+}
+
+build {
+    system -W ${worksrcpath} "GOPATH=${workpath} \
+      TAG=${github.version} make istioctl"
```

Comment:
Very minor, please adjust the spacing for this line to match the mode line. So line it up with the previous line.

Reply:
Done, thanks.